src/views/index/home.njk

```diff
@@ -35,7 +35,7 @@
     
     <p class="govuk-body">{{ i18n.startTimeout }}</p>
     
-    <button class="govuk-button govuk-button--start" data-module="govuk-button" data-event-id="START NOW">
+    <button class="govuk-button govuk-button--start" data-module="govuk-button" data-goal-id="15">
```

Comment:
Goal is recorded using the following onClick="_paq.push(['trackGoal', goalId])"

Reply:
Done